src/roiextractors/extractors/caiman/caimansegmentationextractor.py

```diff
@@ -75,10 +76,16 @@ def _trace_extractor_read(self, field):
         if field in self._dataset_file["estimates"]:
             return lazy_ops.DatasetView(self._dataset_file["estimates"][field]).lazy_transpose()
 
-    def _summary_image_read(self):
+    def _correlation_image_read(self):
         if self._dataset_file["estimates"].get("Cn"):
             return np.array(self._dataset_file["estimates"]["Cn"])
 
+    def _summary_image_read(self):
+        if self._dataset_file["estimates"].get("b"):
+            FOV_shape = self._dataset_file["params"]["data"]["dims"][()]
+            b_sum = self._dataset_file["estimates"]["b"][:].sum(axis=1)
+            return np.array(b_sum).reshape(FOV_shape, order="F")
```

Comment:
Are you able to confirm why the column order is Fortran-like? Is that documented somewhere?

Reply:
trial and error with the dataset you provided plus the following:

https://github.com/flatironinstitute/CaImAn/blob/80e1681bbce8fb4a36b04c57d9e42ffb75a32d58/caiman/base/rois.py#L461
https://github.com/flatironinstitute/CaImAn/blob/80e1681bbce8fb4a36b04c57d9e42ffb75a32d58/caiman/utils/utils.py#L267
https://github.com/flatironinstitute/CaImAn/blob/80e1681bbce8fb4a36b04c57d9e42ffb75a32d58/caiman/utils/visualization.py#L106

Reply:
OK thanks for thoroughly checking this!